install-local-dev-2.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+cd backends/stable_diffusion_torch/
+
+echo ">> Installing pytorch"
+conda install pytorch torchvision torchaudio -c pytorch-nightly
+
+echo ">> Downloading custom python dependencies, might have to pass (i) to ignore existing packages"
+pip install -r requirements.txt
+
+cd ../..
+
+echo ">> Setting up electron app"
+cd electron_app
+npm i
```

Comment:
npm does not ship with macOS. Consider checking for the existence of these commands with e.g.

```shell
if ! [ -x "$(command -v npm)" ]; then
  echo 'Error: npm is not installed.' >&2
  exit 1
fi
```